client/src/App.jsx

```diff
@@ -1,12 +1,19 @@
-import './App.css'
+import "./App.css";
+import { Routes, Route } from "react-router-dom";
+import StudentRegistration from "./registration/studentregistration";
+import TeacherRegistration from "./components/TeacherRegistration";
 
 function App() {
-
   return (
-    <>
-      This is the app
-    </>
-  )
+    <div>
+      <Routes>
+        <Route path="/" element={<div>This is the app</div>} />
+        <Route path="/studentreg" element={<StudentRegistration />} />
+        <Route path="/teacherreg" element={<TeacherRegistration />} />
```

Comment:
These routes should be `teacher/registration` and `student/registration`. Make sure to use nested routers. Here is a [resource](https://stackoverflow.com/a/25305272)